README.md

```diff
@@ -40,13 +40,13 @@ Requests related to the funtionality or usability of the UI can be submitted as
     yarn start
     ```
 4. Testing:
-    
```

Comment:
I don't know why it made these changes, but it seems to just be white space.

Reply:
Hmm. Can you revert these lines to just avoid extra diff?

Reply:
Done